.github/workflows/publish-to-npm.yml

```diff
@@ -53,7 +53,7 @@ jobs:
     with:
       ctc: false # We've been told we don't have to care about this until someone makes us care.
       sign: true
-      tag: latest-beta-rc # Publish as a release candidate, so we can do our validations against it.
+      tag: latest-alpha-rc # Publish as a release candidate, so we can do our validations against it.
```

Comment:
Releasing as `latest-alpha-rc` and `latest-alpha`. I'm not exactly sure what this will do to `latest` since there technically isn't one yet, but we can find out when we publish, right?